### PR TITLE
Chainsaw Sounds

### PR DESCRIPTION
**Pull request by Jaygriff**
:cl: Jaygriff
soundadd: Add sounds for chainsaw startup
/:cl:

[]: # Per #336

### CODE REVIEW

File: code/game/objects/items/weapons/twohanded.dm [hunk 1]
```diff
@@ -471,6 +472,8 @@
 	force = on ? force_on : initial(force)
 	throwforce = on ? force_on : initial(force)
 	icon_state = "chainsaw_[on ? "on" : "off"]"
+	if(on)
```
Comment (JohnGinnane) [inline code/game/objects/items/weapons/twohanded.dm:475]: This isn't necessary as the wield() proc already does this

**Review by JohnGinnane — CHANGES_REQUESTED** [review]

**Review by Jaygriff — COMMENTED** [review]
We only want the sound to play when the chainsaw is turned on. Chainsaw cannot be unwielded as it is a forced two handed weapon. Doesnt use default wield and unwield procs because of this.

**Comment by JohnGinnane** [comment]
Oh right, chainsaw doesn't use wield/unwield. Did you test that it works like that?

**Comment by Jaygriff** [comment]
Yep. Just plays the sound when the chainsaw is toggled on. I do not recall us having a sound for it turning off and I couldn't find it in the files if we did.

**Review by JohnGinnane — APPROVED** [review]

**Comment by JohnGinnane** [comment]
I don't either, cool

**Pull request merged**